ci/kokoro/windows/build-cmake-dependencies.ps1

```diff
@@ -16,6 +16,7 @@
 
 # Stop on errors. This is similar to `set -e` on Unix shells.
 $ErrorActionPreference = "Stop"
+. ci/kokoro/windows/lib/vcpkg-cache.ps1
```

Comment:
Perhaps these lines deserve to be offset from the "Stop on errors" block?

Reply:
Done.